Content.Server/Disease/DiseaseSystem.cs

```diff
@@ -0,0 +1,420 @@
+using System.Threading;
+using Content.Shared.Disease;
+using Content.Shared.Disease.Components;
+using Content.Server.Disease.Components;
+using Content.Server.Clothing.Components;
+using Content.Shared.MobState.Components;
+using Content.Shared.Examine;
+using Content.Shared.Inventory;
+using Content.Shared.Interaction;
+using Content.Server.Popups;
+using Content.Server.DoAfter;
+using Robust.Shared.Player;
+using Robust.Shared.Prototypes;
+using Robust.Shared.Random;
+using Robust.Shared.Serialization.Manager;
+using Content.Shared.Inventory.Events;
+using Content.Server.Nutrition.EntitySystems;
+
+namespace Content.Server.Disease
+{
+
+    /// <summary>
+    /// Handles disease propagation & curing
+    /// </summary>
+    public sealed class DiseaseSystem : EntitySystem
+    {
```

Comment:
these get a bit long so you might want to look into making this a partial class for readability (see `BotanySystem` for an example), not too bad though